### PR TITLE
chore: enable latency based throttling veneer backend

### PR DESCRIPTION
**Pull request by igorbernstein2**
This depends on the latest version off veneer, so #2945 must be merged first

### CODE REVIEW

File: bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/wrappers/veneer/BigtableHBaseVeneerSettings.java [hunk 1]
```diff
@@ -15,12 +15,14 @@
  */
 package com.google.cloud.bigtable.hbase.wrappers.veneer;
 
+import static com.google.cloud.bigtable.config.BulkOptions.BIGTABLE_BULK_THROTTLE_TARGET_MS_DEFAULT;
```
Comment (kolea2) [inline bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/wrappers/veneer/BigtableHBaseVeneerSettings.java:18]: can you also add a small test asserting the default is used when not specified?

**Review by kolea2 — CHANGES_REQUESTED** [review]

File: bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/wrappers/veneer/BigtableHBaseVeneerSettings.java (hunk 1 above)
Comment (mutianf) [reply bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/wrappers/veneer/BigtableHBaseVeneerSettings.java:18]: +1

**Review by mutianf — APPROVED** [review]
LGTM

File: bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/wrappers/veneer/BigtableHBaseVeneerSettings.java (hunk 1 above)
Comment (igorbernstein2) [reply bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/wrappers/veneer/BigtableHBaseVeneerSettings.java:18]: done

**Pull request merged**